### PR TITLE
replace liveblog top ad test with feature switch

### PR DESCRIPTION
**Pull request by dskamiotis**
## What is the value of this and can you measure success?
We are happy with the rendering of a new liveblog-top slot in liveblog in [DCR PR](https://github.com/guardian/dotcom-rendering/pull/11730) 
In addition the page config set up along with Experiments in [frontend PR](https://github.com/guardian/frontend/pull/27259)

## What does this change?
We would like to remove the Experiment and make available to all in prod, with the addition of a feature switch.

## Screenshots
Screenshot 2024-07-08 at 16 37 26
| window.guardian.config      | window.guardian.config.switches      |
|-------------|------------|
| Screenshot 2024-07-10 at 10 51 35 | Screenshot 2024-07-10 at 10 53 44 |

| Top Ad   | 
|-------------|
| Screenshot 2024-07-10 at 10 56 27 |

Screenshot 2024-07-08 at 16 42 18

## Checklist

- [x] Tested locally, and on CODE if necessary
- [x] Will not break dotcom-rendering
- [x] Will not break our database – if updating CAPI, [updated and committed the database files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)
- [ ] Meets our accessibility [standards](https://github.com/guardian/recommendations/blob/e647ef695199ea3116ea20d827ef0f1364270a39/accessibility.md)
  - [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
  - [x] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#Keyboard)
  - [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#colour)

### CODE REVIEW

File: common/app/experiments/Experiments.scala [hunk 1]
```diff
@@ -23,22 +22,13 @@ object ActiveExperiments extends ExperimentsDefinition {
 
 object UseSourcepointPropertyId
     extends Experiment(
-      name = "use-sourcepoint-property-id",
+      name = "use-sourcepoint-property-idDSTEST",
```
Comment (emma-imber) [inline common/app/experiments/Experiments.scala:25]:
```suggestion
      name = "use-sourcepoint-property-id",
```

There should be no diff here

File: common/app/conf/switches/FeatureSwitches.scala [hunk 2]
```diff
@@ -309,6 +309,16 @@ trait FeatureSwitches {
     exposeClientSide = false,
   )
 
+  val LiveBlogTopSponsorshipSwitch = Switch(
+    SwitchGroup.Feature,
+    "live-blog-top-sponsorship",
+    "When on allows new sposorship ad to be displayed on live blogs",
+    owners = Seq(Owner.withGithub("dotcom.platform@theguardian.com")),
```
Comment (emma-imber) [inline common/app/conf/switches/FeatureSwitches.scala:316]:
```suggestion
    owners = Seq(Owner.withGithub("commercial.dev@theguardian.com")),
```

File: common/app/experiments/Experiments.scala (hunk 1 above)
Comment (dskamiotis) [reply common/app/experiments/Experiments.scala:25]: Oh no one of my local testing strategies - thanks Emma

File: common/app/conf/switches/FeatureSwitches.scala [hunk 3]
```diff
@@ -309,6 +309,16 @@ trait FeatureSwitches {
     exposeClientSide = false,
   )
 
+  val LiveBlogTopSponsorshipSwitch = Switch(
+    SwitchGroup.Feature,
+    "live-blog-top-sponsorship",
+    "When on allows new sposorship ad to be displayed on live blogs",
```
Comment (arelra) [inline common/app/conf/switches/FeatureSwitches.scala:315]:
```suggestion
    "When on allows a sponsorship ad to be displayed on live blogs",
```

File: common/app/conf/switches/FeatureSwitches.scala [hunk 4]
```diff
@@ -309,6 +309,16 @@ trait FeatureSwitches {
     exposeClientSide = false,
   )
 
+  val LiveBlogTopSponsorshipSwitch = Switch(
```
Comment (arelra) [inline common/app/conf/switches/FeatureSwitches.scala:312]: Might this switch be better placed in `CommercialSwitches.scala` ?

**Comment by Jakeii** [comment]
This also needs to be merged https://github.com/guardian/frontend/pull/27279

File: common/app/conf/switches/CommercialSwitches.scala [hunk 5]
```diff
@@ -7,6 +7,16 @@ import conf.switches.SwitchGroup.{Commercial, CommercialPrebid, Membership}
 
 trait CommercialSwitches {
 
+  val LiveBlogTopSponsorshipSwitch = Switch(
+    Commercial,
+    "live-blog-top-sponsorship",
+    "When on allows a sposorship ad to be displayed on live blogs",
+    owners = group(Commercial),
+    safeState = On,
```
Comment (SiAdcock) [inline common/app/conf/switches/CommercialSwitches.scala:15]: Nit: I'd say the safe state should be off for this

**Review by SiAdcock — APPROVED** [review]

**Pull request merged**